nifi-nar-bundles/nifi-standard-services/nifi-distributed-cache-services-bundle/nifi-distributed-cache-server/src/main/java/org/apache/nifi/distributed/cache/server/AbstractCacheServer.java

```diff
@@ -177,7 +177,11 @@ public void stop() throws IOException {
         logger.info("Stopping CacheServer {}", new Object[]{this.identifier});
 
         if (serverSocketChannel != null && serverSocketChannel.isOpen()) {
-            serverSocketChannel.close();
+            try {
+                serverSocketChannel.close();
+            } catch (IOException e) {
+                logger.warn(e.getMessage());
```

Comment:
Recommend adding a message to this log:
```suggestion
                logger.warn("Server Socket Close Failed", e);
```